src/Testing/MakesGraphQLRequests.php

```diff
@@ -157,13 +162,19 @@ protected function introspectByName(string $path, string $name): ?array
 
     /**
      * Return the full URL to the GraphQL endpoint.
+     *
+     * @param  array<string, string>  $routeParams  Route params to pass to the Laravel route
      */
-    protected function graphQLEndpointUrl(): string
+    protected function graphQLEndpointUrl(array $routeParams = []): string
     {
         $config = Container::getInstance()->make(ConfigRepository::class);
         $routeName = $config->get('lighthouse.route.name');
+        $mergedRouteParams = array_merge(
+            $config->get('lighthouse.route.test.params') ?? [],
+            $routeParams
+        );
```

Comment:
I don't see why this would be necessary

Reply:
The idea behind it was that you can set global config route params instead of setting it in every testcase and still have the option to add test based route params.

Reply:
Let's hold off on this for now. Is is also quite easy to do make a wrapper function.

Reply:
Okay, i removed this